### ISSUE TITLE
can't render latex expressions and local images

### ISSUE THREAD

**Issue by matteomanuelli** [LolipopJ/mdimg#10]
```
<!-- input.md -->
$\frac{1}{2}$
![test](test.png)
```
```
mdimg -i input.md -o output.png -w 1000
```

output:
![output](https://github.com/user-attachments/assets/545b1a85-ce86-4afb-bcfd-6cc33ae02ce4)

**Comment by LolipopJ** [comment]
Thanks for your feedback!

I released [v1.4.0](https://www.npmjs.com/package/mdimg/v/1.4.0) just now. LaTeX and local files can display correctly in my machine. Hava a try!

If there are further problems, please let me know.

**Comment by matteomanuelli** [comment]
LaTex expressions still are not rendered. Local images are ok

**Comment by LolipopJ** [comment]
> @matteomanuelli LaTex expressions still are not rendered. Local images are ok

Seems like MathJax [doesn't render](https://tex.stackexchange.com/questions/27633/mathjax-inline-mode-not-rendering) inline LaTeX by default.

If I follow the guide above to enable inline rendering, the common `$` will need be replaced with `\$`. It leads to a bad behavior.

I'm thinking of a better solution...

**Comment by LolipopJ** [comment]
@matteomanuelli I released [v1.4.1](https://www.npmjs.com/package/mdimg/v/1.4.1) and you can configuration the behavior of MathJax now.

Please read this [chapter](https://github.com/LolipopJ/mdimg?tab=readme-ov-file#mathjax) to make inline `$` sign enabled. A rendering result can be found [here](https://github.com/LolipopJ/mdimg/blob/main/docs/darwin/github.png).

**Comment by matteomanuelli** [comment]
It works as expected ;)
